### PR TITLE
Fix `destination-exasol` build and CatalogDefinition generation

### PR DESCRIPTION
**Pull request by evantahler**
Fixes the `destination-exasol` build (added via https://github.com/airbytehq/airbyte/pull/21200).  Lots of stuff changed when we ripped apart our monorepo 😅 

Tested locally via 

```
./gradlew :airbyte-integrations:connectors:destination-exasol:test :airbyte-integrations:connectors:destination-exasol:integrationTest
```

### CODE REVIEW

**Comment by evantahler** [comment]
Debugging:

`SUB_BUILD=ALL_CONNECTORS ./gradlew :airbyte-config:specs:generateOssConnectorCatalog` does add the new connector
`SUB_BUILD=ALL_CONNECTORS ./gradlew :airbyte-config:init:test` fails the test AND rebuilds the file, removing the new connector

**Review by rodireich — APPROVED** [review]

File: airbyte-config/specs/build.gradle [hunk 1]
```diff
@@ -21,16 +21,6 @@ task generateSeedConnectorSpecs(type: JavaExec, dependsOn: compileJava) {
     args new File(project(":airbyte-config:init").projectDir, '/src/main/resources/seed')
 }
 
-task downloadCombinedConnectorCatalog(type: JavaExec, dependsOn: compileJava) {
```
Comment (bnchrch) [inline airbyte-config/specs/build.gradle:24]: @pedroslopez Sanity check, we can delete all the download logic in connectors. Correct?

Comment (pedroslopez) [reply airbyte-config/specs/build.gradle:24]: Correct!

**Review by bnchrch — APPROVED** [review]
@evantahler Should be good to merge 👍

**Comment by evantahler** [comment]
Thank you @bnchrch and @pedroslopez!

**Pull request merged**